modeling/dataset/samplers/next_item_prediction.py

```diff
@@ -141,7 +138,7 @@ def get_user_id_to_index_cross_domain_mapping(self):
 
     @classmethod
     def create_from_config(cls, config, **kwargs):
-        domains = [config['target_domain']] + ast.literal_eval(config['other_domains'])
+        domains = [kwargs['target_domain']] + kwargs['other_domains']
```

Comment:
For me it is also a suitable solution, however, I prefer adding this params in the configs (as I wrote above)